src/char/int_auction.cpp

```diff
@@ -242,12 +242,12 @@ void inter_auctions_fromsql(void)
 		for( i = 0; i < MAX_SLOTS; i++ )
 		{
 			Sql_GetData(sql_handle, 15 + i, &data, NULL);
-			item->card[i] = atoi(data);
+			item->card[i] = strtoul(data, NULL, 10);
 		}
 
 		for (i = 0; i < MAX_ITEM_RDM_OPT; i++) {
 			Sql_GetData(sql_handle, 15 + MAX_SLOTS + i*3, &data, NULL);
-			item->option[i].id = atoi(data);
+			item->option[i].id = strtoul(data, NULL, 10);
```

Comment:
This causes a warning upon compiling. (possible loss of data)

![image](https://user-images.githubusercontent.com/48939874/85685366-90c68100-b701-11ea-9e6c-0b0c613da024.png)

Reply:
Interesting, this shouldn't have been changed, since random option IDs haven't changed.
I'll fix